docs/developer-docs/smart-contracts/advanced-features/periodic-tasks.mdx

```diff
@@ -360,29 +413,20 @@ from kybra.canisters.management import management_canister
 
```

Comment:
There imports here have a couple of things not used in the example that could be cleaned up, it not out of the scope of this change